docs/Examples.md

```diff
@@ -0,0 +1,107 @@
+In the following sections, we present example configurations for a variety of Git Providers and Issue Trackers combinations.
+
+## Bitbucket & Jira
```

Comment:
In the examples is necessary to add some more info, like tell user about failure exporter needed default labels ?